__tests__/app.test.js

```diff
@@ -158,3 +158,201 @@ describe("tests endpoint api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("tests POST endpoint api/articles/:articleid/comments", () => {
+  test("POST: 201 responds with a new comment when posting a comment from an existing user, to an existing article with no comments and increases comment count of the correct article", () => {
+    const testComment = {
+      username: "rogersop",
+      comment: "I like it a bit, but not loads",
+    };
+    const countComments = () => {
+      return request(app).get("/api/articles/2/comments");
+    };
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send(testComment)
+      .expect(201)
+      .then(({ body: { returnedComment } }) => {
+        expect(returnedComment).toMatchObject({
+          article_id: 2,
+          author: "rogersop",
+          body: "I like it a bit, but not loads",
+          comment_id: 19,
+          created_at: expect.any(String),
+        });
+      })
+      .then(() => {
+        return countComments();
+      })
+      .then(({ body: { comments } }) => {
+        expect(comments.length).toBe(1);
+      });
+  });
+  test("POST: 201 responds with a new comment when posting a comment from an existing user, to an existing article which has comments already, and increases comment count of the correct article", () => {
+    const testComment = {
+      username: "icellusedkars",
+      comment: "I like this one more",
+    };
+
+    const countComments = () => {
+      return request(app).get("/api/articles/1/comments");
+    };
```

Comment:
You don't need to refactor your code to this (unless you want to!), but have a think about how you could put this util function (👀 ) in scope for all the tests so you don't have to redefine it each time 😄